File: AtelierMock/C#/CaractereCopieur/Copieur.cs
namespace CaractereCopieur
{
    public class Copieur
    {
        private Interfaces.ISource _source;
        private Interfaces.IDestination _destination;

        public Copieur(Interfaces.ISource source, Interfaces.IDestination destination)
        {
            _source = source;
            _destination = destination;
        }


        public void Copier()
        {
        }
    }
}

File: AtelierMock/VB.Net/CaractereCopieur/Copieur.vb
Public Class Copieur
    Private aSource As Interfaces.ISource
    Private aDestination As Interfaces.IDestination
    
    Public Sub New(source As Interfaces.ISource, destination As Interfaces.IDestination)
        aSource = source
        aDestination = destination
    End Sub

    Public Sub Copier()
        Throw New NotImplementedException()
    End Sub
End Class
